app/controllers/ConfirmYourPostcodeController.scala

```diff
@@ -130,12 +128,13 @@ class ConfirmYourPostcodeController @Inject()(
             Redirect(routes.SendLetterErrorController.onPageLoad(mode))
           case TechnicalIssueResponse(responseStatus, responseMessage) =>
             auditService.findYourNinoTechnicalError(personalDetailsResponse, personalDetails, responseStatus, responseMessage)
-            Redirect(routes.TechnicalErrorController.onPageLoad())
+            Redirect(routes.TechnicalLetterErrorController.onPageLoad())
```

Comment:
Only a suggestion however, I would consider renaming to 'LetterTechnicalErrorController' as its still a technical error but is specific to the letter.